cmd/expiration-mailer/main.go

```diff
@@ -168,13 +174,13 @@ func (m *mailer) findExpiringCertificates() error {
 			 AND cert.expires > :cutoffA
 			 AND cert.expires <= :cutoffB
 			 AND cs.status != "revoked"
-			 AND cs.lastExpirationNagSent <= :nagCutoff
+			 AND COALESCE(TIMESTAMPDIFF(SECOND, cs.lastExpirationNagSent, cert.expires) > :nagCutoff, 1)
```

Comment:
`0000-00-00 00:00:00` isn't a real date (there is no zeroth month), so `TIMESTAMPDIFF` returns NULL for that too. The "no nag sent at all yet" test fails without the `COALESCE`.